### PR TITLE
[SPARK-20041][DOC] Update docs for NaN handling in approxQuantile

### PR DESCRIPTION
**Pull request by zhengruifeng**
## What changes were proposed in this pull request?
Update docs for NaN handling in approxQuantile.

## How was this patch tested?
existing tests.

### CODE REVIEW

**Comment by zhengruifeng** [comment]
retest this please

File: R/pkg/R/stats.R [hunk 1]
```diff
@@ -149,7 +149,8 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
 #' This method implements a variation of the Greenwald-Khanna algorithm (with some speed
 #' optimizations). The algorithm was first present in [[http://dx.doi.org/10.1145/375663.375670
 #' Space-efficient Online Computation of Quantile Summaries]] by Greenwald and Khanna.
-#' Note that rows containing any NA values will be removed before calculation.
+#' Note that NA values will be ignored in numerical columns before calculation. For
+#'   columns only containing Na values, an empty list is returned.
```
Comment (gatorsmile) [inline R/pkg/R/stats.R]: `Na` -> `NA`

**Comment by gatorsmile** [comment]
LGTM except a typo.

File: R/pkg/R/stats.R (hunk 1 above)
Comment (zhengruifeng) [inline R/pkg/R/stats.R]: Good catch. Updated.

**Comment by gatorsmile** [comment]
Thanks! Merging to master.

**Pull request closed without merging**